src/components/TabSelect/_TabButton.js

```diff
@@ -15,8 +15,8 @@ const Styled = component => styled(component)`
   cursor: pointer;
   margin-bottom: -1px;
   margin-right: 5px;
-  padding: 10px 20px;
-  font-size: ${props => props.theme.fontSizes.large};
+  padding: ${props => (props.small ? '5px 10px' : '10px 20px')};
+  font-size: ${props => props.theme.fontSizes[props.small ? 'small' : 'large']};
```

Comment:
I think we should use `normal` font size with `props.small`, also I'd rather do something like

```
font-size: ${props => props.small ? props.theme.fontSizes.normal : props.theme.fontSizes.large};
```

so that those are easier to search and detect, e.g. see https://github.com/weaveworks/service-ui/pull/2803.